docs/src/configuration.md

```diff
@@ -69,6 +69,37 @@ and the user-provided one are used in the user configuration if
 `stylix.base16Scheme` is not changed in the user config. If that is the case,
 only the user override is used.
 
+### Extending
+
+When passing colors to unsupported targets, it is possible to access values from the configured
```

Comment:
```suggestion
If you are creating custom modules, it is possible to access values from the configured
```

Reply:
Some people prefer creating exports local to their config. I feel like limiting this suggestion to stylix modules might be deterring for them, which is why I phrased it like that.

Reply:
That would be fine too.